lib/src/base-styles/_nx-btn.scss

```diff
@@ -79,24 +79,24 @@
 }
 
 .nx-btn--error {
-  background-color: var(--nx-swatch-red-45);
-  border-color: var(--nx-swatch-red-45);
+  background-color: var(--nx-swatch-red-40);
+  border-color: var(--nx-swatch-red-40);
   color: var(--nx-swatch-white);
 
   &:hover, &:focus, &:active {
```

Comment:
Shouldn't `:focus` and `:active` be removed from this selector?

Reply:
> Shouldn't `:focus` and `:active` be removed from this selector?

Yes, they were there to share the `color` attribute, but that's not even necessary. https://github.com/sonatype/sonatype-react-shared-components/pull/481/commits/1ddadb97ab08d82edf313cd1de4f4c1a392ea935